spec/index.html

```diff
@@ -536,6 +569,70 @@ <h3>RDF Documents and Syntaxes</h3>
       and different ordering of triples. While these aspects can have great
       effect on the convenience of working with the <a>RDF document</a>,
       they are not significant for its meaning.</p>
+
+    <p>The basis for <a>concrete RDF syntaxes</a> is the structure of the
+      <a href="#data-model">abstract data model</a>,
+      called the <dfn>abstract syntax</dfn>;
+      summarized in the following two tables.</p>
+    <table id="tab-graph-abstract-syntax" class="simple">
+      <caption>The abstract syntax of <a>RDF graphs</a></caption>
+      <tr>
+        <th>Production</th>
+        <th>Defined as</th>
+      </tr>
+      <tr>
+        <td>RDF graph</td>
+        <td>a <i>set</i> of <i>zero or more</i> <a>triples</a></td>
+      </tr>
+      <tr>
+        <td>triple</td>
+        <td>a <i>3-tuple</i> of
+          a <a>subject</a>, a <a>predicate</a>, and an <a>object</a></td>
```

Comment:
It might be worthwhile to say somewhere that `an alternate labeling of this <i>3-tuple</i> is as an <a>entity</a>, an <a>attribute</a>, and a <a>value</a>`, and that `a <a>predicate</a> or <a>attribute</a> can also be referred to as a <a>property</a>`

Reply:
Section 3.1 calls them subject-predicate-object, _not_ entity-attribute-value. 

Attribute-value is particular has other meanings e.g. XML.
"value" is used in many ways.

> <a>entity</a>

Where is entity `<dfn>`?

`<a>entity</a>` would be a link to `#dfn-entity` (handled by reSpec).

Reply:
`<a>subject</a>` `<a>predicate</a>` and `<a>object</a>` link to the formal definitions in section 3.

"entity-attribute-value" is not formal terminology. 

"Value" is unhelpful due to its multiple meanings (maybe you are getting cross-over from SHACL" where "value" is used for object but that not in RDF Concepts).

"attribute" mentioned in RDF Concept is that in XML / HTML.

Reply:
> Any explanation of the graph-based abstract data model in relation to E/R (conceptual) or EAV (storage?) notions

Maybe in the primer, but not in RDF Concepts. ("Why stop at E/R, EAV?")

Keep RDF Concepts about what RDF is with the terminology RDF uses.

In 1.2 it's quoted `"entity"` and AFAICT because semantics uses the word (and then only twice in explanatory text). The 1.2 clause could even be removed (not this PR!).